plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/controller/Users.js

```diff
@@ -290,6 +290,7 @@ Ext.define('NX.coreui.controller.Users', {
         callback: cb
       });
     }
+    me.callParent();  // triggers transition between list/detail view
```

Comment:
@djsauble this makes the transition happen as expected, would love some feedback if there's a smrter way to accomplish this
Problem appears to have been simply that loadStore() was overridden for this controller, preventing any of the Drilldown parent logic from firing